### ISSUE TITLE
Don't link to the code-review frontend in the Phabricator comment

### ISSUE THREAD

**Issue by marco-c** [mozilla/code-review#1016]
Link to Phabricator directly instead, with a link to the diff analyzed by the bot.

**Comment by EvaBardou** [comment]
@marco-c For this part:
> with a link to the diff analyzed by the bot.

isn't it duplicated from #451? Should I just remove the two links in my PR and let Valentin add the new link in his PR?

**Comment by marco-c** [comment]
Yeah, #451 was about adding the Phab link only, without removing the other two. This is about removing the other two and adding the Phab one.
They could be implemented in the same PR or in separate PRs, if in separate PRs we should fix #451 first.

**Comment by EvaBardou** [comment]
Ok, I will let Valentin tackle his part and I will work on this next
